addon/components/fabulous-table.js

```diff
@@ -66,12 +66,16 @@ export default Ember.Component.extend({
      * @returns {boolean}
      */
     isScrolledToBottom() {
-        let scrollContainerHeight = this.get('scrollContainer').outerHeight();
-        let topOffsetOfScrollContainer = this.get('scrollContainer').offset().top;
-        let bottomSelector = this.get('noSpinner') ? '.fabulous-row' : '.spinner';
-        let topOffsetOfLastItem = this.$(bottomSelector).last().offset().top;
-        
-        return topOffsetOfLastItem - topOffsetOfScrollContainer <= scrollContainerHeight;
+        if (this.$(bottomSelector).length > 0) {
+            let scrollContainerHeight = this.get('scrollContainer').outerHeight();
+            let topOffsetOfScrollContainer = this.get('scrollContainer').offset().top;
+            let bottomSelector = this.get('noSpinner') ? '.fabulous-row' : '.spinner';
+            let topOffsetOfLastItem = this.$(bottomSelector).last().offset().top;
+
+            return topOffsetOfLastItem - topOffsetOfScrollContainer <= scrollContainerHeight;
+        } else {
+            return false;
+        }
```

Comment:
`bottomSelector` is not defined at this point. Do it like this:

```suggestion
        const bottomSelector = this.get('noSpinner') ? '.fabulous-row' : '.spinner';

        if (this.$(bottomSelector).length < 1) {
            return false;
        }
        
        const scrollContainerHeight = this.get('scrollContainer').outerHeight();
        const topOffsetOfScrollContainer = this.get('scrollContainer').offset().top;
        const topOffsetOfLastItem = this.$(bottomSelector).last().offset().top;

        return topOffsetOfLastItem - topOffsetOfScrollContainer <= scrollContainerHeight;
```